src/grandorgue/GOOrganController.cpp

```diff
@@ -686,6 +690,33 @@ bool GOOrganController::IsToImportCombinationsFor(
   return isToImport;
 }
 
+static std::vector<char> loadFileBytes(const wxString &file) {
```

Comment:
```suggestion
static std::vector<char> load_file_bytes(const wxString &fileName) {
```
due the GO naming conventions about out-of-class functions